github-actions/bazel/setup/add-repository-cache.sh

```diff
@@ -0,0 +1,6 @@
+echo """
+build --repository_cache=~/.cache/bazel_repo_cache
```

Comment:
Nit: We can share this using an environment variable to not repeat it twice.

Reply:
Unfortunately for our case here `env` values are not shared between "steps" in composite actions